components/style/values/specified/mod.rs

```diff
@@ -7,7 +7,7 @@ use cssparser::{self, Parser, ToCss, Token};
 use euclid::size::Size2D;
 #[cfg(feature = "gecko")]
 use gecko_bindings::ptr::{GeckoArcPrincipal, GeckoArcURI};
-use parser::{ParserContext, ParserContextExtraData};
```

Comment:
I think you’ll want `#[cfg(feature = "gecko")] use parser::ParserContextExtraData;` here.